src/Common/tests/Common.Tests.csproj

```diff
@@ -15,6 +15,7 @@
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="$(CommonTestPath)\System\Collections\DictionaryExtensions.cs" />
```

Comment:
And elsewhere. The problem is only when a file is not below the project file in the tree.

---
In reply to: [107041630](https://github.com/dotnet/corefx/pull/17201#discussion_r107041630) [](ancestors = 107041630)